src/routes/_layout.svelte

```diff
@@ -27,5 +27,7 @@
 <!-- <Nav {segment}/> -->
 
 <main>
+  <svelte:component this={Nav}></svelte:component>
```

Comment:
Segment isn't being passed, so it's always `undefined` inside of the component.
```suggestion
  <Nav  {segment} />
```